### PR TITLE
feat: add blog post with 5.0.6 details

### PR DESCRIPTION
**Pull request by bsmth**

### CODE REVIEW

File: blog/2021-02-05-questdb-release-5-0-6-postgres-wire.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+This language feature adds more flexibility in cases where `CREATE TABLE`
+queries are run and tables already exist:
+
+```questdb-sql
+CREATE TABLE IF NOT EXISTS my_table (ts TIMESTAMP, value INT) timestamp(ts);
+```
+
+More full examples using this statement in multiple languages can be found on
+the
+[Insert data](http://localhost:3000/docs/develop/insert-data#postgres-compatibility)
```
Comment (mpsq) [inline blog/2021-02-05-questdb-release-5-0-6-postgres-wire.md:95]:
```suggestion
[Insert data](/docs/develop/insert-data#postgres-compatibility)
```

File: blog/2021-02-05-questdb-release-5-0-6-postgres-wire.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+[query Data](/docs/develop/query-data#postgres-compatibility) using popular
+tools and frameworks in Node, Go, Rust, Java and Python.
+
+## Grafana alerts
+
+Support for the RFC339Nano timestamp format enables the use of alerting via
+Grafana. This helps identify unintended changes, minimize disruptions and stay
+on top of your system or infrastructure:
+
+![Alerts being triggered in Grafana based on data in QuestDB](/img/blog/2021-02-05/grafana_alerts.png)
```
Comment (mpsq) [inline blog/2021-02-05-questdb-release-5-0-6-postgres-wire.md:71]: Could we rename the files using kebab-case?

**Review by mpsq — CHANGES_REQUESTED** [review]
minor tweaks

File: blog/2021-02-05-questdb-release-5-0-6-postgres-wire.md (hunk 1 above)
Comment (bsmth) [reply blog/2021-02-05-questdb-release-5-0-6-postgres-wire.md:95]: oops 👍🏻

File: blog/2021-02-05-questdb-release-5-0-6-postgres-wire.md (hunk 2 above)
Comment (bsmth) [reply blog/2021-02-05-questdb-release-5-0-6-postgres-wire.md]: done

**Review by mpsq — APPROVED** [review]

**Pull request merged**